lib/cn/cn_tree_iter.h

```diff
@@ -23,16 +23,6 @@ cn_tree_walk_callback_fn(
     struct cn_tree_node *node,
```

Comment:
Not yet, I'll take another look at it, see if we can move that stuff elsewhere..  I think it was intended to avoid exposing cn-tree internals, but no one can really use it without pulling in cn_tree_internal.h, so...